data/fetch.jl

```diff
@@ -28,7 +28,7 @@ function fetch_pretalx(filter_fn::Function, resource; out_file=resource, token=n
     []
   end
   resp = HTTP.get(
-    "https://pretalx.com/api/events/juliacon-2022/$(resource)?limit=500",
+    "https://pretalx.com/api/events/juliacon2023/$(resource)?limit=500",
```

Comment:
Is this juliacon-2023, not juliacon2023

Reply:
No, its `juliacon2023`  unfortunately. Complain to the whoever set this up.